tfjs-core/src/backends/cpu/backend_cpu.ts

```diff
@@ -3442,6 +3446,59 @@ export class MathBackendCPU implements KernelBackend {
     }
   }
 
+  private fft2dBatch(x: Tensor3D): Tensor3D {
```

Comment:
If the input length is the exponent of 2, I think we can use `fftRadix2` which is faster by making use of the divide and conquer approach.

Reply:
yes, that would be faster, I can try to add that :)

Reply:
I changed it so that `fft2d` falls back to using `fft` across rows then columns, which should end up using `fftRadix2` if the input length is exponent of 2.

Reply:
I removed the use of `fftRadix2`, it's much slower (~x10) than the naïve `fft2d`. I can try to implement an `fft2dRadix2` implementation, but that might take me awhile.